### PR TITLE
Clipboard message fixed

### PR DESCRIPTION
**Pull request by tommasini**
**Description**

The clipboard message is fixed

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #https://github.com/MetaMask/mobile-planning/issues/255

### CODE REVIEW

File: app/components/Views/RevealPrivateCredential/index.js [hunk 1]
```diff
@@ -338,9 +338,7 @@ class RevealPrivateCredential extends PureComponent {
 
 		const msg =
 			privateCredentialName === PRIVATE_KEY
-				? `${strings(`reveal_credential.${privateCredentialName}_copied`)}\n${strings(
-						`reveal_credential.${privateCredentialName}_copied_time`
-				  )}`
+				? `${strings(`reveal_credential.${privateCredentialName}_copied`)}`
```
Comment (sethkfman) [inline app/components/Views/RevealPrivateCredential/index.js:341]: Remove conditional and add helper text to Android users 

`Device.isIos() ? strings('reveal_credential.${privateCredentialName}_copied_time') : 'Remember to clear your clipboard after you have completed your action.'`

Comment (tommasini) [reply app/components/Views/RevealPrivateCredential/index.js]: Done it

**Comment by sethkfman** [comment]
The team met today and discussed the options for the clipboard message. It was decided to remove the 1 minute message and continue with further investigation and discussion.

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**